src/App.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(
+    (movie) => movie.title
+      .toLocaleLowerCase().includes(query.trim().toLocaleLowerCase())
+      || movie.description
+        .toLowerCase().includes(query.trim().toLocaleLowerCase()),
```

Comment:
It would be great to create variables for `query.trim().toLocaleLowerCase()` and other formatted values to make your code cleaner